src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceOutputFormatProvider.java

```diff
@@ -34,17 +36,25 @@
    */
   public SalesforceOutputFormatProvider(SalesforceSinkConfig config) {
     ImmutableMap.Builder<String, String> configBuilder = new ImmutableMap.Builder<String, String>()
-      .put(SalesforceConstants.CONFIG_USERNAME, config.getUsername())
-      .put(SalesforceConstants.CONFIG_PASSWORD, config.getPassword())
-      .put(SalesforceConstants.CONFIG_CONSUMER_KEY, config.getConsumerKey())
-      .put(SalesforceConstants.CONFIG_CONSUMER_SECRET, config.getConsumerSecret())
-      .put(SalesforceConstants.CONFIG_LOGIN_URL, config.getLoginUrl())
       .put(SalesforceSinkConstants.CONFIG_SOBJECT, config.getSObject())
       .put(SalesforceSinkConstants.CONFIG_OPERATION, config.getOperation())
       .put(SalesforceSinkConstants.CONFIG_ERROR_HANDLING, config.getErrorHandling().getValue())
       .put(SalesforceSinkConstants.CONFIG_MAX_BYTES_PER_BATCH, config.getMaxBytesPerBatch().toString())
       .put(SalesforceSinkConstants.CONFIG_MAX_RECORDS_PER_BATCH, config.getMaxRecordsPerBatch().toString());
 
+    OAuthInfo oAuthInfo = config.getOAuthInfo();
+    if (oAuthInfo != null) {
+      configBuilder.put(SalesforceConstants.CONFIG_OAUTH_TOKEN, oAuthInfo.getAccessToken());
+      configBuilder.put(SalesforceConstants.CONFIG_OAUTH_INSTANCE_URL, oAuthInfo.getInstanceURL());
```

Comment:
nit: use same style (chained or not of builder calls in both `if` and `else`

Reply:
fixed